### DISCUSSION TITLE
The existing data in the database has been lost, and I want to rewrite the saved data.

### DISCUSSION THREAD

**Discussion by wooriki** [supabase/supabase#18403] [Contribute to Supabase]
Hello, I'm junior developer

Until last week, the project (service) was running properly, but from this week or the weekend, all the data in the database,
including user login information, has disappeared.

I would like to reuse the 'The_Summer_We' database, which is currently suspended.
How can I do that?
I upgraded to a pro account today and have made the payment.
In a hurry, I created a new one, but the existing DB was completely wiped, and user information was lost.
The DB categories, including Authentication, have been reset.

I've been in a rush and left inquiries here and there. 
I would like to request a recovery. Please help me to use it again.

Thank you.

**Comment by KevinBrolly** [comment]
> I've been in a rush and left inquiries here and there.
> I would like to request a recovery. Please help me to use it again.

Hey @wooriki, I can see your multiple open support tickets - please refrain from opening multiple issues or issues in multiple places as this only divides our response and makes it less likely that you will get assistance quickly. The official support cannel is going to be the best place to get help with your issue as it relates directly to your account, whereas this is a forum intended for the community to help each other with 

I am closing this issue in favour of your already opened tickets, please follow up there. Thanks.

**Reply by wooriki** [reply]
Thank you. So, would it be better to delete the most recently recreated project?

I have translated it into English for you.
